internal/api/export/sample/public.pem

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
these don't belong in '/internal'

I think these belong in /examples
https://github.com/golang-standards/project-layout#examples

Reply:
I think `/examples` is usually reserved for _Go_ examples. I agree this shouldn't be in internal though. Maybe `/testdata`?